controllers/staff_controller.py

```
@@ -1,15 +1,20 @@
 from flask import Blueprint, request, jsonify
-from models import Staff
 from extensions import db
 from repositories.staff_repository import StaffRepository
+from repositories.schedule_repository import ScheduleRepository
 from services.staff_service import StaffService
+from services.schedule_service import ScheduleService
+from models import Staff, Shift
```

Ruff (0.8.2)

`models.Staff` imported but unused

Remove unused import

(F401)